### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,40 +1,43 @@
...
+* Publish and run Move smart contracts
+* Program with Sui objects
+* Run a Sui Fullnode
+* Use Sui Explorer to see transactions
+
+See instructions for all of the above at:
+https://docs.sui.io
 
 In the coming months, we will release:
-1. A public devnet that allows Sui devs to do all of the above on a shared network powered by Mysten-operated authorities
```
Comment (randall-Mysten) [inline ROADMAP.md:9]: this is not rendering as a list in preview, and starts with 2. 

image

Comment (randall-Mysten) [reply ROADMAP.md:9]: fixed with a new commit

**Review by randall-Mysten — APPROVED** [review]

**Review by bwann52 — COMMENTED** [review]
looks accurate to me!
wanted to ask, is this the main/only roadmap we currently have documented?

**Comment by Clay-Mysten** [comment]
> looks accurate to me! wanted to ask, is this the main/only roadmap we currently have documented?

Yep. @sblackshear put that together in the run-up to the original launch.

**Pull request merged**